package.json

```diff
@@ -810,6 +810,35 @@
           "command": "cmake.outline.cleanRebuildAll",
           "when": "resourceFilename == CMakeLists.txt && cmake:enableFullFeatureSet"
         }
+      ],
+      "touchBar": [
+        {
+          "command": "cmake.configure",
+          "when": "!cmake:enableFullFeatureSet",
+          "group": "CMake:build"
+        },
+        {
```

Comment:
The condition negation is probably a typo here, right?  However, we want various commands (like quickStart, all configure operations) to be always on because they represent a way back from a partial to a full activation mode. I think you can remove the condition here.

Reply:
Is quickStart needed when you already have configured (or even at least ready to be configured) project?

TouchBar is actually quite small, and placing custom commands there should have reasoned advantage of having it there at the moment: overextending commands and groups are just cut from Touch Bar, no slide gestures supported.

I can try to use images instead of default text title - but it [requires copies of already existing SVGs to be stored in PNG format](https://code.visualstudio.com/updates/v1_17#_touch-bar-support). So I need your advice here: should we do this?

There's another way to place more commands - `"alt"` property. It was supposed to switch command on Alt pressed - but I haven't noticed it's working. Just in case I'll check it again :)

Reply:
Quick start probably isn't needed for already configured projects, but it's possible someone deletes files and wants to start over.

"Always on" was referring to having the commands be available in VS Code, not the Touchbar, I think.  We're not asking for more commands in the Touchbar.